draft-ietf-rtcweb-jsep.xml

```diff
@@ -2581,6 +2586,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t>The "a=rtcp-mux" line MUST only be added if
                         present in the most recent answer.</t>
 
+			<t>An "a=rtcp-mux-only" line, as specified in
```

Comment:
I don't think we want to fail things that use rtcp-mux but not rtcp-mux-only. That's worse than the current situation.

I agree we should fail if the policy is set to require but the other side doesn't say rtcp-mux; this is covered in 4.1.1, but could be made more clear in Section 5 discussion of handling a remote description.